frontend/src/pages/BYONImages/BYONImagesTable.tsx

```diff
@@ -330,6 +331,17 @@ export const BYONImagesTable: React.FC<BYONImagesTableProps> = ({ images, forceU
                       alignItems={{ default: 'alignItemsCenter' }}
                     >
                       <FlexItem>{image.name}</FlexItem>
+                      <FlexItem>
+                        <ResourceNameTooltip
+                          resource={{
+                            apiVersion: '',
+                            metadata: {
+                              name: image.id,
+                            },
+                            kind: 'ImageStream',
+                          }}
+                        />
+                      </FlexItem>
```

Comment:
Why not wrap these two together? Seems odd to change the functionality of `ResourceNameTooltip` because you don't want to wrap the value?

Reply:
Oh i see use the children prop. Yeah, I went with this method to preserve the spacing in the flex box, bc there is an error that sometimes will show

Reply:
Keep the two FlexItems that predate you -- wrap the `image.name`